python/tox.ini

```diff
@@ -43,12 +43,19 @@ commands =
     # flake8 includes black check due to flake8-black
     # flake8 includes isort check which checks for import order due to flake8-isort
     flake8 pynessie tests tools
-    # ignore https://pyup.io/v/51457/f17 -> https://github.com/pytest-dev/py/issues/287
-    safety check --ignore=51457
     pylint --jobs=0 pynessie tests tools
     mypy --install-types --non-interactive -p pynessie -p tests -p tools
 
 
+[testenv:safety]
+basepython = python
+deps = safety
+commands =
+    # using separate env because of https://github.com/pyupio/safety/issues/455
+    # note that requirements_lint.txt imports all other requirement files
+    safety check --file requirements_lint.txt
```

Comment:
from CI run:
```
     Safety v2.3.5 is scanning for Vulnerabilities...
    Scanning dependencies in your files:
  
    -> requirements_lint.txt
  
    Using non-commercial database
    Found and scanned 39 packages
    Timestamp 2023-03-31 07:50:58
    0 vulnerabilities found
    0 vulnerabilities ignored
  +==============================================================================+
  
   No known security vulnerabilities found. 
```

this might exclude scanning of transitive dependencies, but i guess its still better than flat out removing safety until they manage to release their new version (that blocks our other upgrade PRs)